queue_services/account-mailer/src/account_mailer/worker.py

```diff
@@ -48,7 +48,8 @@
 from account_mailer.enums import Constants, MessageType, SubjectType, TemplateType, TitleType
 from account_mailer.services import minio_service  # pylint: disable=wrong-import-order
 from account_mailer.services import notification_service  # pylint: disable=wrong-import-order
-from account_mailer.utils import format_currency, get_local_formatted_date  # pylint: disable=wrong-import-order
+from account_mailer.utils import format_day_with_suffix  # pylint: disable=wrong-import-order
```

Comment:
any idea why we're using disable=wrong-import-order?

Reply:
no idea, I probably copy and pasted it from another line but I will remove all of them and fix any linting issues

Reply:
removed the pylint hints and cleaned up the imports